src/Aspire.Hosting.Azure.Sql/Aspire.Hosting.Azure.Sql.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you revert this?  `git checkout <sha-before-your-changes> -- src/Aspire.Hosting.Azure.Sql/Aspire.Hosting.Azure.Sql.csproj`